js/views/userPageVw.js

```diff
@@ -307,6 +307,25 @@ module.exports = baseVw.extend({
           self.model.set({ownPage: self.options.ownPage});
           self.render();
           !self.currentItemHash && self.loadingDeferred.resolve();
+          
+          // Handle was requested
+          if (self.model.get('page').profile.handle) {
+            var profile = self.model.get('page').profile,
```

Comment:
There already is a `profile` variable declared in this same scope, a few lines above. I think it might correspond to the same thing you need. If so, please use it, otherwise please rename your variable.

On a different note, I know we're not 100% consistent in this, but I think we should be making our variable declarations at the top of each scope, both to avoid situations like this and also to accurately mimic variable hoisting. @jjeffryes What do you think - are you fine with us adopting that convention (at least going forward)?

If you do indeed want block level scope, feel free to use ES6's `let`, which is supported as long as this module is in strict mode (ensure there is a `use strict;` declaration up top).